### PR TITLE
Play/pause video playback on spacebar press

### PR DESCRIPTION
**Pull request by samchrisinger**

### CODE REVIEW

File: exp-player/addon/components/exp-video.js [hunk 1]
```diff
@@ -11,6 +13,31 @@ export default ExpFrameBaseComponent.extend(FullScreen, MediaReload, {
     displayFullscreen: true,  // force fullscreen for all uses of this component, always
     fullScreenElementId: 'player-video',
 
+    spaceHandler: null,
+    didRender() {
```
Comment (chrisseto) [inline exp-player/addon/components/exp-video.js]: Why not use `didInsertElement`

Comment (samchrisinger) [inline exp-player/addon/components/exp-video.js]: Can you please explain or point me to a reference explaining why this might be better?

Comment (chrisseto) [inline exp-player/addon/components/exp-video.js]: `didInsertElement` will only run once. This has the potential to run multiple times.

Comment (samchrisinger) [inline exp-player/addon/components/exp-video.js]: If the component re-renders might it be a good idea to re-bind event listeners (especially with `player` closured in).

**Pull request merged**